spec/features/admin/merchants/index_spec.rb

```diff
@@ -1,9 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe "Admin Merchants Index" do
-  let!(:merchant1) {FactoryBot.create(:merchant)}
-  let!(:merchant2) {FactoryBot.create(:merchant)}
-  let!(:merchant3) {FactoryBot.create(:merchant)}
+  let!(:merchant1) {FactoryBot.create(:merchant, name: "Jurassic Quark")}
+  let!(:merchant2) {FactoryBot.create(:merchant, name: "A Parfait Storm")}
+  let!(:merchant3) {FactoryBot.create(:merchant, name:"Fry Hard", status: 1)}
```

Comment:
Having one initialize enabled makes testing where it filters to much easier I'm sure.  I think I tried changing its status during the test and it just wasn't picking up the change so this is a great idea